test/framework/deprecated.go

```diff
@@ -17,22 +17,37 @@ limitations under the License.
 package framework
 
 import (
+	"bytes"
 	"context"
 	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
 	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-
 	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/cluster-api/test/framework/management/kind"
+	"sigs.k8s.io/cluster-api/test/framework/options"
+
+	corev1 "k8s.io/api/core/v1"
```

Comment:
corev1 and bootstrapv1 are duplicate imports

Reply:
fixed!